<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorProgressBarTest</class>
 <widget class="QWidget" name="ColorProgressBarTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多彩进度条</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ColorProgressBar" name="colorProgressBar1" native="true"/>
   </item>
   <item>
    <widget class="QSlider" name="slider1">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorProgressBar" name="colorProgressBar2" native="true"/>
   </item>
   <item>
    <widget class="QSlider" name="slider2">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorProgressBar" name="colorProgressBar3" native="true"/>
   </item>
   <item>
    <widget class="QSlider" name="slider3">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorProgressBar</class>
   <extends>QWidget</extends>
   <header>colorprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
